doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3236,6 +3236,11 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
+.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

Comment:
Is this issue applicable for 2.5.x as well?
And also, does it also affect HCI LE Create Connection command or it only happens when Host sends Extended command?

Reply:
Yes it affects 2.5x as well. Will add that

And yes HCI LE Create Conenction is also relevant as it is directed to the Extended Create command